rollup.config.js

```diff
@@ -0,0 +1,72 @@
+import babel from 'rollup-plugin-babel';
+import commonjs from 'rollup-plugin-commonjs';
+import replace from 'rollup-plugin-replace';
+import resolve from 'rollup-plugin-node-resolve';
+import uglify from 'rollup-plugin-uglify';
+
+import pkg from './package.json';
+
+function distBuild(options = {}) {
+    return {
+        input: 'src/index.js',
+        output: {
+            file: `dist/${options.filename}`,
+            format: options.format,
+            name: 'aphrodite',
+            sourcemap: options.sourcemap,
+        },
+        plugins: [
+            babel({
+                exclude: ['node_modules/**'],
+            }),
+            replace({
+                'process.env.NODE_ENV': JSON.stringify('production'),
+            }),
+            resolve({
+                browser: true,
+            }), // so rollup can find node modules
+            commonjs(), // so rollup can convert node modules to ESM if needed
+            options.minify && uglify(),
+        ]
+    };
+}
+
+function standardBuilds(filename) {
+    return {
+        input: `src/${filename}.js`,
+        external: [
+            ...Object.keys(pkg.dependencies),
+            'inline-style-prefixer/static/createPrefixer',
+            'inline-style-prefixer/static/plugins/calc',
+            'inline-style-prefixer/static/plugins/crossFade',
+            'inline-style-prefixer/static/plugins/cursor',
+            'inline-style-prefixer/static/plugins/filter',
+            'inline-style-prefixer/static/plugins/flex',
+            'inline-style-prefixer/static/plugins/flexboxIE',
+            'inline-style-prefixer/static/plugins/flexboxOld',
+            'inline-style-prefixer/static/plugins/gradient',
+            'inline-style-prefixer/static/plugins/imageSet',
+            'inline-style-prefixer/static/plugins/position',
+            'inline-style-prefixer/static/plugins/sizing',
+            'inline-style-prefixer/static/plugins/transition',
+        ],
+        output: [
+            { file: `lib/${filename}.js`, format: 'cjs' },
+            { file: `lib/${filename}.mjs`, format: 'es' },
```

Comment:
This will build a version that does not compile `import` to `require`, which enables build tools that can understand `import` (e.g. webpack 2+) to perform additional optimizations such as "tree-shaking" and scope hoisting.

```sh
~/aphrodite ❯❯❯ head lib/index.js
'use strict';

Object.defineProperty(exports, '__esModule', { value: true });

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var createPrefixer = _interopDefault(require('inline-style-prefixer/static/createPrefixer'));
var calc = _interopDefault(require('inline-style-prefixer/static/plugins/calc'));
var crossFade = _interopDefault(require('inline-style-prefixer/static/plugins/crossFade'));
var cursor = _interopDefault(require('inline-style-prefixer/static/plugins/cursor'));

~/aphrodite ❯❯❯ head lib/index.mjs
import createPrefixer from 'inline-style-prefixer/static/createPrefixer';
import calc from 'inline-style-prefixer/static/plugins/calc';
import crossFade from 'inline-style-prefixer/static/plugins/crossFade';
import cursor from 'inline-style-prefixer/static/plugins/cursor';
import filter from 'inline-style-prefixer/static/plugins/filter';
import flex from 'inline-style-prefixer/static/plugins/flex';
import flexboxIE from 'inline-style-prefixer/static/plugins/flexboxIE';
import flexboxOld from 'inline-style-prefixer/static/plugins/flexboxOld';
import gradient from 'inline-style-prefixer/static/plugins/gradient';
import imageSet from 'inline-style-prefixer/static/plugins/imageSet';
```

The .mjs extension is the proposed way to specify this type of build that Node will eventually adopt, but it is really early to see it in many places. I'd be just as happy to build this somewhere like `es/${filename}.js` if you prefer, and that might be a bit smoother for folks anyway until the dust settles on the .mjs extension.